Tasks/code-push-vsts-task/task.json

```diff
@@ -0,0 +1,117 @@
+    "instanceNameFormat": "Release $(packagePath) to CodePush ($(appName) - $(deploymentName)) ",
+    "inputs": [
+        {
+            "name": "authType",
+            "type": "pickList",
+            "label": "Authentication Method",
+            "defaultValue": "ServiceEndpoint",
+            "helpMarkDown": "",
+            "options": {
+                "ServiceEndpoint": "Service Endpoint",
+                "AccessKey": "Access Key"
+            }
+        },
+        {
+            "name": "serviceEndpoint",
+            "type": "connectedService:codepush-auth-key",
+            "label": "Service Endpoint",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The VSTS Service Endpoint that specifies the identity for Code Push.",
+            "visibleRule": "authType = ServiceEndpoint"
+        },
+        {
+            "name": "accessKey",
+            "type": "string",
+            "label": "Access Key",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Access key used to authenticate with the CodePush service",
+            "visibleRule": "authType = AccessKey"
+        },
+        {
+            "name": "appName",
+            "type": "string",
+            "label": "App Name",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Name of the app you want to release an update for"
+        },
+        {
+            "name": "packagePath",
+            "type": "filePath",
+            "label": "Package Path",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Path to the file or directory that contains the content you want to release"
+        },
+        {
+            "name": "appStoreVersion",
+            "type": "string",
+            "label": "App Store Version",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The app store version that this release depends on (must be semver compliant)"
+        },
+        {
+            "name": "deploymentName",
+            "type": "pickList",
+            "label": "Deployment Name",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Name of the deployment under the app which you want to push the release to",
+            "options": {
+                "Staging": "Staging",
+                "Production": "Production"
+            },
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "description",
+            "type": "string",
+            "label": "Package Description",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Description of the package to be released"
+        },
+        {
+            "name": "isMandatory",
+            "type": "boolean",
+            "label": "Mandatory",
```

Comment:
Are you referencing the name of this or the label? The label is already "Mandatory", and i would actually prefer keeping internal var names isX for boolean, but if you feel strongly about this, i don't mind changing it.